spec/basic/expected_conditions_spec.js

```diff
@@ -1,255 +1,250 @@
-  });
-
-  it('should have titleContains', function() {
-    expect(EC.titleContains('My Angular').call()).toBe(true);
-    expect(EC.titleContains('My AngularJS App').call()).toBe(true);
-  });
-
-  it('should have titleIs', function() {
-    expect(EC.titleIs('My Angular').call()).toBe(false);
-    expect(EC.titleIs('My AngularJS App').call()).toBe(true);
-  });
-
-  it('should have urlContains', function() {
-      var baseUrlFromSpec = browser.baseUrl;
-      expect(EC.urlContains('/form').call()).toBe(true);
-      expect(EC.urlContains(baseUrlFromSpec+ 'index.html#/form').call()).toBe(true);
-  });
-
-  it('should have urlIs', function() {
-      var baseUrlFromSpec = browser.baseUrl;
-      expect(EC.urlIs(browser.baseUrl).call()).toBe(false);
-      expect(EC.urlIs(baseUrlFromSpec+'index.html#/form').call()).toBe(true);
-  });
-
-  it('should have elementToBeClickable', function() {
-    var invalidIsClickable = EC.elementToBeClickable($('#INVALID'));
-    var buttonIsClickable = EC.elementToBeClickable($('#disabledButton'));
-
-    expect(invalidIsClickable.call()).toBe(false);
-    expect(buttonIsClickable.call()).toBe(true);
-    element(by.model('disabled')).click();
-    expect(buttonIsClickable.call()).toBe(false);
-  });
-
-  it('should have textToBePresentInElement', function() {
-    var invalidHasText = EC.textToBePresentInElement($('#INVALID'), 'shouldnt throw');
-    var hideableHasText = EC.textToBePresentInElement($('#shower'), 'Shown');
-
-    expect(invalidHasText.call()).toBe(false);
-    expect(hideableHasText.call()).toBe(true);
-    element(by.model('show')).click();
-    expect(hideableHasText.call()).toBe(false);
-  });
-
-  it('should have textToBePresentInElementValue', function() {
-    var invalid = $('#INVALID');
-    var about = element(by.model('aboutbox'));
-
-    expect(EC.textToBePresentInElementValue(invalid, 'shouldnt throw').call()).toBe(false);
-    expect(EC.textToBePresentInElementValue(about, 'text box').call()).toBe(true);
-  });
-
-  it('should have elementToBeSelected', function() {
-    var checkbox = element(by.model('show'));
-
-    expect(EC.elementToBeSelected(checkbox).call()).toBe(true);
-    checkbox.click();
-    expect(EC.elementToBeSelected(checkbox).call()).toBe(false);
-  });
-
-  it('should have not', function() {
-    var presenceOfValidElement = EC.presenceOf($('#shower'));
-    expect(presenceOfValidElement.call()).toBe(true);
-    expect(EC.not(presenceOfValidElement).call()).toBe(false);
-  });
-
-  it('should have and', function() {
-    var presenceOfValidElement = EC.presenceOf($('#shower'));
-    var presenceOfInvalidElement = EC.presenceOf($('#INVALID'));
-    var validityOfTitle = EC.titleIs('My AngularJS App');
-
-    expect(EC.and(presenceOfValidElement, validityOfTitle).call()).toBe(true);
+const EC = protractor.ExpectedConditions;
+
+describe('expected conditions', () => {
+  beforeEach(async () => {
+    await browser.get('index.html#/form');
+  });
+  
```

Comment:
Please remove spaces.

Reply:
Hrrmmm... it looks like this doesn't show it. If you look at your diff, you will see what I'm seeing. https://github.com/angular/protractor/pull/5013/files